db-scheduler-boot-starter/src/main/java/com/github/kagkarlsson/scheduler/boot/config/DbSchedulerProperties.java

```diff
@@ -106,6 +107,17 @@
     @NotNull
     private Duration shutdownMaxWait = SchedulerBuilder.SHUTDOWN_MAX_WAIT;
 
+    /**
+     * <p>Which log level to use when logging task failures. Defaults to {@link Level#DEBUG}.</p>
+     */
+    @NotNull
+    private Level failureLogLevel = Level.DEBUG;
+
+    /**
+     * <p>Whether or not to log the {@link Throwable} that caused a task to fail.</p>
+     */
+    private boolean logStackTraceOnFailure = false;
+
```

Comment:
These need to refer to the constants in the `SchedulerBuilder` to ensure we get the same default